packages/rainbowkit/src/components/Avatar/Avatar.tsx

```diff
@@ -45,17 +45,12 @@ export function Avatar({ address, imageUrl, loading, size }: AvatarProps) {
       >
         <AvatarComponent address={address} ensImage={imageUrl} size={size} />
       </Box>
-      {typeof loading === 'boolean' && (
+      {loading && (
         <Box
           color="accentColor"
           display="flex"
           height="full"
           position="absolute"
-          style={{
```

Comment:
You need to add these styles back, or no?

Reply:
Ah yeah. They are not needed anymore since they are only used for transition effects